FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries/abstractions/platform/freertos_plus/src/iot_pkcs11_mbedtls.c

```diff
@@ -773,66 +1414,79 @@ CK_DECLARE_FUNCTION(CK_RV, C_GetFunctionList)
  *                              to the list.
  *
  * @return CKR_OK if successful.
- * Else, see <a href="https://tiny.amazon.com/wtscrttv">PKCS #11
- * specification</a> for more information.
+ * Else, see <a href="https://tiny.amazon.com/wtscrttv">PKCS #11 specification</a>
+ * for more information.
  */
-/* @[declare pkcs11_mbedtls_c_getslotlist] */
-CK_DECLARE_FUNCTION(CK_RV, C_GetSlotList)
-(CK_BBOOL xTokenPresent, CK_SLOT_ID_PTR pxSlotList, CK_ULONG_PTR pulCount)
+/* @[declare_pkcs11_mbedtls_c_getslotlist] */
+CK_DECLARE_FUNCTION( CK_RV, C_GetSlotList )( CK_BBOOL xTokenPresent,
+                                             CK_SLOT_ID_PTR pxSlotList,
+                                             CK_ULONG_PTR pulCount )
 { /*lint !e9072 It's OK to have different parameter name. */
     CK_RV xResult = CKR_OK;
 
     /* Since the mbedTLS implementation of PKCS#11 does not depend
      * on a physical token, this parameter is ignored. */
-    (void)(xTokenPresent);
+    ( void ) ( xTokenPresent );
 
-    if (PKCS11_MODULE_IS_INITIALIZED != CK_TRUE) {
+    if( PKCS11_MODULE_IS_INITIALIZED != CK_TRUE )
+    {
         xResult = CKR_CRYPTOKI_NOT_INITIALIZED;
     }
 
-    if (NULL == pulCount) {
+    if( NULL == pulCount )
+    {
         xResult = CKR_ARGUMENTS_BAD;
     }
 
-    if (xResult == CKR_OK) {
-        if (NULL == pxSlotList) {
+    if( xResult == CKR_OK )
```

Comment:
Minor and fyi... I would stick to one style, either var==condition or condition==var. Some prefer later, since that's safer even == is mistakenly changed to single =.